coops-ui/src/util/config-helper.ts

```diff
@@ -0,0 +1,42 @@
+import axios from '@/axios-auth'
+
+export default {
+  /**
+   * fetch config from API
+   */
+  fetchConfig () {
+    const url = '/config/configuration.json'
+    const headers = {
+      'Accept': 'application/json',
+      'ResponseType': 'application/json',
+      'Cache-Control': 'no-cache'
+    }
+
+    return axios
+      .get(url, { headers })
+      .then(response => {
+        const apiUrl = response.data['API_URL']
+        axios.defaults.baseURL = apiUrl
+        console.log('Set axios.defaults.baseURL to: ' + apiUrl)
+
+        const authUrl = response.data['AUTH_URL']
+        sessionStorage.setItem('AUTH_URL', authUrl)
+        console.log('Set uthUrl to: ' + authUrl)
+
+        const authApiUrl = response.data['AUTH_API_URL']
+        sessionStorage.setItem('AUTH_API_URL', authApiUrl)
+        console.log('Set authApiUrl to: ' + authApiUrl)
+
+        const payApiUrl = response.data['PAY_API_URL']
+        sessionStorage.setItem('PAY_API_URL', payApiUrl)
+        console.log('Set payApiUrl to: ' + payApiUrl)
+
+        const addressCompleteKey = response.data['ADDRESS_COMPLETE_KEY']
+        window['addressCompleteKey'] = addressCompleteKey
+        console.log('Set addressCompleteKey')
+      })
+      .catch(err => {
```

Comment:
Can this catch be removed? Otherwise everything looks good.

Reply:
Yes, if it removed then the error bubbles up.

In ticket 913 I'm reworking some of the data loading and I have removed the catch from here. I have added a catch in main.ts instead.